getting-started/vs-integration/new-project-wizard.md

```diff
@@ -12,6 +12,8 @@ This article demonstrates how to use the Telerik Visual Studio extensions to cre
 
 To create a new Telerik UI for Blazor application, use the Create New Project wizard. The wizard detects all installed versions of Telerik UI for Blazor and lists them in the **Version** combobox&mdash;this feature enables you to start your project with the desired version. To make sure the components are up to date, you can also [get the latest version]({%slug getting-started-vs-integration-latest-version%}).
```

Comment:
* The Style Guide recommends using bold formatting for the names of wizards (see [here](https://docs.telerik.com/style-guide/element-formatting#guidelines)).
* When used as an adjective , up-to-date should be hyphenated.

```suggestion
To create a new Telerik UI for Blazor application, use the **Create New Project** wizard. The wizard detects all installed versions of Telerik UI for Blazor and lists them in the **Version** combobox to let you to start your project with the desired version. To make sure the components are up-to-date, you can also [get the latest version]({%slug getting-started-vs-integration-latest-version%}).
```